tests/transport/TestMsg/TestMsg.h

```diff
@@ -0,0 +1,58 @@
+#ifndef OPENDDS_TRANSPORT_TEST_MSG_H
+#define OPENDDS_TRANSPORT_TEST_MSG_H
+
+#include "TestMsg_Export.h"
+
+#include <dds/DCPS/Serializer.h>
+#include <dds/DCPS/TypeSupportImpl.h>
+
+struct TestMsg_Export TestMsg {
+  ACE_CDR::ULong key;
+  TAO::String_Manager value;
+
+  TestMsg();
+  TestMsg(ACE_CDR::ULong msgKey, TAO::String_Manager msgValue);
```

Comment:
Should this just be `const char*` or `std::string`?  It's odd to see `String_Manager` as a parameter type.

Reply:
Yeah, not sure, just copied it as is from one of the tests. Changed it to `const char*`.